styles/reset.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
```

Comment:
Do you need this commented code?

Reply:
I have used the reset.css as is from [http://meyerweb.com/eric/tools/css/reset/](http://meyerweb.com/eric/tools/css/reset/) , so I thought it's important to save information about source and license.

But now I deleted most part of rules in this file, so now this comment seems unnecessary in any case. 
[Fixed](https://github.com/js-training-sep-2020/hw-1/pull/5/commits/9946871b741dbec80dd6a92a2a0632ac7f24d949)

Reply:
Good, thanks!